docs/source/contents/4_draft_mapping.rst

```diff
@@ -1777,10 +1777,651 @@ Decision
 
 **Do not migrate!**
 
-
 subject
 =======
 
```

Comment:
This is really silly for my to really only catch this, but looks like we need the predicate table for this still.

Reply:
Thanks - I've just added a table.